helpme/tests.py

```diff
@@ -170,37 +170,37 @@ def test_comment_creation(self):
 
 
     def test_teams_view_support(self):
-        uri = reverse("helpme:team-list")
+        uri = reverse("helpme_admin:team-list")
         response = self.client.get(uri)
 
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "My Teams")
-        self.assertContains(response, '<a href="/support/teams/')
+        self.assertContains(response, '<a href="/administration/support/teams/')
```

Comment:
With checking endpoints, I'll often reverse() the URL name rather than hard code it.  It make the tests a little more flexible if the path changes since all we really want to just make sure it's present.

Reply:
@renderbox Thanks for the suggestion- I will update the tests!